.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+# Title
+
+Please write a descriptive title that describes
+what you've fixed (or created) with this Pull Request.
+
+# Summary
+
+Fixes #Issue_Number
+
+>note: The "Fixes #nnn" syntax in the PR description causes
+>GitHub to close the issue when this PR is merged. See 
+> [this GitHub help document](https://help.github.com/articles/closing-issues-via-commit-messages/)
+> for details. It saves all of us some work. Not all PRs need to reference an issue.
+>See [our contributing guide](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md) for details.
+
+Insert short (less than 140 character) summary here.
```

Comment:
Do we have to say here? I'd also move this up before the fixes notation.
Maybe just say: Provide a short description of what you've done.